spec.bs

```diff
@@ -837,6 +854,9 @@ partial dictionary SharedStorageRunOperationMethodOptions {
 };
 </xmp>
 
+The {{SharedStorageWorkletGlobalScope/privateAggregation}} getter steps are to
```

Comment:
Shall we link getter steps?

Reply:
Actually do we have to explicitly define the getter steps given that we already mention it should be set to [=get the privateAggregation=]?

Reply:
Linked getter steps for the two monkeypatches. I think unfortunately we probably have to do this as I don't think the Private Aggregation spec 'controls' the Shared Storage and Protected Audience getter steps (at least after the monkey patches are incorporated properly). It's a little weird that we instruct how implementers should expose their objects, but I think it's probably the layering that makes the most sense.

Reply:
Yeah that makes sense. It’s a little weird, but not too bad.